src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/express/ExpressCaseServiceTest.java

```diff
@@ -0,0 +1,175 @@
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NatureApplication.PENSION_ATTACHMENT_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NatureApplication.PENSION_COMPENSATION_ATTACHMENT_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NatureApplication.PENSION_COMPENSATION_SHARING_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NatureApplication.PENSION_SHARING_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NatureApplication.PERIODICAL_PAYMENT_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NatureApplication.PROPERTY_ADJUSTMENT_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NottinghamCourt.BOSTON_COUNTY_COURT_AND_FAMILY_COURT;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NottinghamCourt.NOTTINGHAM_COUNTY_COURT_AND_FAMILY_COURT;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.Region.MIDLANDS;
+
+class ExpressCaseServiceTest {
+
+    @Mock
+    private FinremCaseData caseData;
+
+    @BeforeEach
+    public void setUp() {
+        expressCaseService = new ExpressCaseService();
+        expressCaseService.expressCaseFrcs = List.of("FR_s_NottinghamList_1", "FR_s_NottinghamList_2");
+    }
+
+    @InjectMocks
+    private ExpressCaseService expressCaseService;
+
+    @Test
+    void shouldEnrolledInExpressPilot_WhenCaseDataMeetsRequirements() {
+
+        FinremCaseData caseData = FinremCaseData.builder()
+            .regionWrapper(RegionWrapper.builder()
+                .allocatedRegionWrapper(AllocatedRegionWrapper.builder()
+                    .courtListWrapper(DefaultCourtListWrapper.builder()
+                        .nottinghamCourtList(NOTTINGHAM_COUNTY_COURT_AND_FAMILY_COURT)
+                        .build())
+                    .regionList(MIDLANDS)
+                    .midlandsFrcList(RegionMidlandsFrc.NOTTINGHAM)
+                    .build())
+                .build())
+            .natureApplicationWrapper(NatureApplicationWrapper.builder()
+                .natureOfApplicationChecklist(List.of(
+                    MAINTENANCE_PENDING_SUIT,
+                    LUMP_SUM_ORDER,
+                    PROPERTY_ADJUSTMENT_ORDER,
+                    A_SETTLEMENT_OR_A_TRANSFER_OF_PROPERTY,
+                    PERIODICAL_PAYMENT_ORDER,
+                    PENSION_SHARING_ORDER,
+                    PENSION_COMPENSATION_SHARING_ORDER,
+                    PENSION_ATTACHMENT_ORDER,
+                    PENSION_COMPENSATION_ATTACHMENT_ORDER
+                ))
+                .build())
+            .fastTrackDecision(YesOrNo.NO)
+            .estimatedAssetsChecklistV2(UNDER_TWO_HUNDRED_AND_FIFTY_THOUSAND_POUNDS)
+            .build();
+
+        expressCaseService.setExpressCaseEnrollmentStatus(caseData);
+        assertEquals(ENROLLED, caseData.getExpressCaseParticipation());
+    }
```

Comment:
Close.  You're missing that the application type needs to be matrimonial on these tests.